src/handlers/comment/handlers/first.ts

```diff
@@ -25,7 +29,15 @@ export const verifyFirstCheck = async (): Promise<void> => {
       const isFirstComment = resp.data.filter((item) => item.user?.login === payload.sender.login).length === 1;
       if (isFirstComment) {
         //first_comment
-        const msg = `${generateHelpMenu()}\n@${payload.sender.login}`;
+        if (firstTimeContributorHeader) {
+          msg += `${firstTimeContributorHeader}\n`;
+        }
+        if (firstTimeContributorHelpMenu) {
+          msg += `${generateHelpMenu()}\n@${payload.sender.login}\n`;
```

Comment:
If I read the issue comments correctly, we don't want to tag the user? @pavlovcik

Reply:
I think tagging the user is nice to get their attention for this. Just make sure it's in normal text, and not monospace or else the tag doesn't work.